README.md

```diff
@@ -74,7 +75,32 @@ documentation for more details.
   Environment
   Bootstrapping](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.pipelines-readme.html#cdk-environment-bootstrapping)
   for details.
-  
+
+## Developer Workflow
```

Comment:
How about a section of "Initial setup", where you recommend they deploy the OIDC provider?